features/facebookImport/src/model/analysis.js

```diff
@@ -217,9 +219,8 @@ export async function runAnalysis(analysisClass, enrichedData) {
     }
 }
 
-export async function analyzeFile(file, facebookAccount) {
-    const zipFile = new ZipFile(file, window.pod);
-    const enrichedData = { ...file, zipFile, facebookAccount };
+export async function analyzeZip(zipData, zipFile, facebookAccount, pod) {
+    const enrichedData = { ...zipData, zipFile, facebookAccount, pod };
```

Comment:
It makes sense it goes here, but it's even more data added to an object of which we don't have a spec. Any alternative you can think of here? There's already redundant information in the shape of the zipData *and* the zipFile

Reply:
The solution here would be the same as for importers. To remove the need to add `...zipData`  to `enrichedData` and get it from the `zipFile`. Will do that in another PR as it might affect other ministories.